demos/java/run_java_on_occlum.sh

```diff
@@ -36,11 +36,9 @@ init_instance() {
 
 build_web() {
     # Copy JVM and JAR file into Occlum instance and build
-    mkdir -p image/usr/lib/jvm
-    cp -r /opt/occlum/toolchains/jvm/java-11-alibaba-dragonwell image/usr/lib/jvm
-    cp /usr/local/occlum/x86_64-linux-musl/lib/libz.so.1 image/lib
-    mkdir -p image/usr/lib/spring
-    cp ../${jar_path} image/usr/lib/spring/
+    rm -rf image
+    cp ../webserver.yaml . 
```

Comment:
Could remove the "cp ../webserver.yaml ."

Reply:
fixed